File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cs
public class Solution {
    public int RemoveDuplicates(int[] nums) {
        var k = 1;
        for(var i=1; i < nums.Length; i++){
            if(nums[i] != nums[i-1])
                nums[k++] = nums[i];
        }
        return k;
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        auto k = 1;
        for(auto i=1; i < nums.size(); i++){
            if(nums[i] != nums[i-1])
                nums[k++] = nums[i];
        }
        return k;
    }
};